homeassistant/components/hue/light.py

```diff
@@ -420,7 +420,10 @@ def device_info(self):
         elif flash == FLASH_SHORT:
             command["alert"] = "select"
             del command["on"]
-        else:
+        elif (
+            not hasattr(self.light, "manufacturername")
```

Comment:
Please follow the same pattern as we have with `is_osram` and `is_philips`

Reply:
thanks for advise. is used in new version.